code/datums/flock/flock.dm

```diff
@@ -28,6 +28,7 @@ var/flock_signal_unleashed = FALSE
 	/// associative list of used names (for traces, drones, and bits) to true values
 	var/list/active_names = list()
 	var/list/enemies = list()
+	var/list/atom/movable/ignores = list()
```

Comment:
Was a little confused, was under the impression that list initialization shouldn't specify the type of thing it holds?

Reply:
why would they not?????????????????????????????????????????????????????????

Reply:
Idk, ignore